lib/widgets/layouts/adaptive_layout/app_adaptive_scaffold_body_view.dart

```diff
@@ -59,6 +59,7 @@ class AppAdaptiveScaffoldBodyView extends StatelessWidget {
         builder: (context, activeNavigationBar, __) {
           return PopScope(
             canPop: activeNavigationBar == AdaptiveDestinationEnum.rooms,
+            // ignore: deprecated_member_use
```

Comment:
Why need to ignore in here?

Reply:
https://docs.flutter.dev/release/breaking-changes/navigator-and-page-api

Reply:
This change need to verified carefully, so i think we merged it later for testing